README.md

```diff
@@ -0,0 +1,74 @@
+khealth
+=====
+
+[![Docker Repository on Quay.io](https://quay.io/repository/coreos/khealth/status "Docker Repository on Quay.io")](https://quay.io/repository/coreos/khealth)
+
+**khealth** is a kubernetes cluster monitoring suite. Routines are defined which exercise various kubernetes subsystems. Collectors receive status update events from Routines and compute current state.
+
+## Quick start
+
+If you have a kubernetes cluster, you can deploy khealth.
+
+```sh
+cd khealth/
+kubectl create -f ./contrib/k8s/khealth-ns.yaml
+kubectl create -f ./contrib/k8s/khealth-{rc,service}.yaml
+```
+
+This will create a nodeport service which exposes the following status endpoints.
+
+| Command  | NodePort |
+| ------------- |:-------------:|
+| rcscheduler  | 31337 |
+
+## Architecture
+
+A module is defined as a single entrypoint command which makes use of a set of routines and a single collector to expose a metric set via the collector's status endpoint.
+
+- **cmd/**:
+
+  This is where the module entrypoint programs live. Each module should have exactly one package in cmd/ which produces an executable which "enacts" that module.
```

Comment:
* `s/module/Module/g`
* "...exactly one `main` package..."? (then you could cut the sentence to: "Each Module should have exactly one `main` package in an eponymous directory beneath `cmd/`." (because `main` implies an executable command, right?)